spyder/api/plugin_registration/confpage.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Spyder completion plugin configuration page."""
+
+# Third party imports
+from qtpy.QtWidgets import (QGroupBox, QVBoxLayout, QCheckBox,
+                            QGridLayout, QLabel)
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.api.preferences import PluginConfigPage
+from spyder.api.plugins import SpyderPlugin
+
+
+class PluginsConfigPage(PluginConfigPage):
+    def setup_page(self):
+        newcb = self.create_checkbox
+        self.plugins_checkboxes = {}
+
+        header_label = QLabel(
+            _("Spyder can run with a reduced number of internal and external "
+              "plugins in order to provide a lighter experience. Any plugin "
+              "unchecked in this page will be unloaded immediately and will "
+              "not be loaded next time Spyder starts."))
```

Comment:
> Here you can turn on/off any internal or external Spyder plugin to disable functionality that you don't use or have a lighter experience. Unchecked plugins in this page will be unloaded immediately and will not be loaded the next time Spyder starts.

What do you think about rewriting this message like this?

Reply:
> Unchecked plugins in this page will be unloaded immediately

This is not true